test/jdk/java/security/cert/CertPathValidator/OCSP/OCSPTimeout.java

```diff
@@ -29,24 +29,24 @@
 
 /**
  * @test
- * @bug 8179502
+ * @bug 8179502 8337826
```

Comment:
You should only include the bugid if the test is testing a specific bug fix. In this case, it is just additional debugging so no need to add it.